activesupport/test/metadata/shared_metadata_tests.rb

```diff
@@ -14,7 +14,9 @@ def test_encryption_and_decryption_with_same_purpose
   end
 
   def test_verifies_array_when_purpose_matches
-    unless null_serializing?
+    if null_serializing?
```

Comment:
I don't like this approach and I would better prefer to not define this test at all if not needed but that would require a bit more refactoring and I was not confident enough doing those changes

Reply:
Would [`skip`](https://www.rubydoc.info/gems/minitest/5.15.0/Minitest%2FAssertions:skip) work here instead of `assert(true)`?  Or would that not have the desired effect in relation to the assertion counters?

Reply:
Oh, yes, of course! `skip` seems to be much more suitable here
Let me quickly do the change
Thanks!